Initial getting started and reusable text blocks for this (#23)
pulsar-edit/pulsar-edit.github.io: common-text-blocks.md

# Reusable reference blocks

This section is for reusable blocks to establish standards for presentation
within our Documentation in regards to our blocks of warnings and other
common areas. These use Vuepress' slots for danger, warning, tip and details.
Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.

## For sections that are under construction.

> ``` ::: warning Under Construction

Reviewer: I'd add a line-break before `::: warning`

# Reusable reference blocks

This section is for reusable blocks to establish standards for presentation
within our Documentation in regards to our blocks of warnings and other
common areas. These use Vuepress' slots for danger, warning, tip and details.
Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.

## For sections that are under construction.

``` ::: warning Under Construction

This document is under construction, please check back soon for updates.
Please see [our socials](link/to/socials/list) and feel free to ask for assistance or enquire as to the status of this document.

:::
```

## For archived sections

```

::: danger STOP

> This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar you use this at your own risk.

Reviewer 2: ```suggestion
This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar, use this at your own risk.
```

# Reusable reference blocks

This section is for reusable blocks to establish standards for presentation
within our Documentation in regards to our blocks of warnings and other
common areas. These use Vuepress' slots for danger, warning, tip and details.

> Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.

Author: I know, but I was stressing the point here that those first two should be avoided at all costs unless they are strictly necessary to use.

> 

Author: I probably did after this at some point.....not sure.

## For sections that are under construction.

> ``` ::: warning Under Construction
> 
> The documentation here is under heavy construction. Updates are coming, and
> likely in the works. If not, please report a issue in our repository for this
> site. You can find on the page linked in our header.
> 
> :::

Reviewer 3: ```suggestion
``` ::: warning Under Construction

This document is under construction, please check back soon for updates.
Please see [our socials](link/to/socials/list) and feel free to ask for assistance or enquire as to the status of this document.

:::
```

I think asking for GH issues to be raised for WIP documents might be soliciting unnecessary issues to be created. If it is being actively worked on then I think it better to come to Discord etc. and ask.
Once there then various people can help answer the question if the info is currently missing from the doc and has the side effect of drawing attention to the doc if it is currently stuck behind a blocker or has stalled.

```

## For archived sections

```

> ::: danger STOP
> 
> This is an archive of the atom documentation. This is only here for historical
> purposes. This is solely a functional reference of how things used to work.
> Processes in this section may have changed in both functionality and
> presentation since these were originally written. Our updated documentation can
> be found at: <insert link here>
> 
> :::

Reviewer 3: My suggestion would be:

```suggestion
::: danger STOP

This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar you use this at your own risk.
Current Pulsar documentation is found at [Generic Link](link/to/main/docs/section).

:::
```

The link wouldn't be specific to link to newer versions etc., it would just take you away from the archive area and into the main docs - for example for people who landed on that page from a google search.

Author: Remove the extra spaces, Not sure why, but it doesn't like blank newlines mixed in with these. Found that out the hard way with my original versions.
